lib/android/ui/ui.nit

```diff
@@ -17,134 +15,123 @@
 # Views and services to use the Android native user interface
 module ui
 
+# Implementation note:
+#
+# We cannot rely on `Activity::on_restore_instance_state` to implement
+# `on_restore_state` is it only invoked if there is a bundled state,
+# and we don't use the Android bundled state.
```

Comment:
Ahhh.. Maybe precise the scope then: `Implementation note for future-me and android experts:`